luatest/server.lua

```diff
@@ -265,6 +243,28 @@ function Server:start(opts)
 
     self:initialize()
 
+    fio.mktree(self.workdir)
```

Comment:
I would like to see directories creation as a separate method to be able to rewrite it in a child class.

Note: It'll require to separate it from the socket path length error.

Reply:
BTW, the socket path length check can be performed at initialization. It is just validation of the given parameters for correctness.

Reply:
> BTW, the socket path length check can be performed at initialization. It is just validation of the given parameters for correctness.

Yes, but in this case we need to parse URI two times. Well, I think it is OK.

Reply:
> I would like to see directories creation as a separate method to be able to rewrite it in a child class.

Fixed.